newIDE/app/src/AssetStore/ExtensionStore/ExtensionInstallDialog.js

```diff
@@ -94,18 +97,29 @@ const ExtensionInstallDialog = ({
   const onInstallExtension = React.useCallback(
     () => {
       if (canInstallExtension) {
-        if (alreadyInstalled) {
-          const answer = Window.showConfirmDialog(
-            'This extension is already in your project, this will install the latest version. You may have to do some adaptations to make sure your game still works. Do you want to continue?'
-          );
```

Comment:
It might be important to keep this message. It was added to avoid extension creators to override the extension update they are working on by mistake.
Maybe, it could detect an upgrade with the same or lower version and change the color of the update button or something like this.